test/unit/test_hash.c

```diff
@@ -148,7 +153,6 @@ void free_kv(HtKv *kv) {
 	Person *p = kv->value;
 	free (p->name);
 	free (p);
-	free (kv);
```

Comment:
This free shuold go here

Reply:
No, given the optimization made, the free(kv) does not need to be done anymore in the free function provided by the user. Which actually makes sense: in RList the user does not provide a free function that frees the ListIter, that's usually freed by the list implementation itself.